core/pkg/opencost/allocation.go

```diff
@@ -311,6 +312,7 @@ func (pvk *PVKey) FromString(key string) error {
 type PVAllocation struct {
 	ByteHours  float64 `json:"byteHours"`
 	Cost       float64 `json:"cost"`
+	Adjustment float64 `json:"adjustment"` //@bingen:field[ignore]
```

Comment:
I would move to the end, in line with @michaelmdresser 's comments above

Reply:
Good point, will do!

Reply:
Done

Reply:
@nikovacevic @ameijer @michaelmdresser I think the added caution is understandable, and I think it's perfectly fine to get into a habbit of just adding new things to the end. However, `@bingen:field[ignore]` actually prevents the fields from appearing on the struct definition that gets passed to the generator, so they can appear in any order. This is the only circumstance where order doesn't matter though